### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by rammazzoti2000**
This branch is used for code review.

This project is trying to replicate as much as possible the NewsWeek.com page.
The objectives for this project were to use as many Bootstrap classes as possible to recreate the website mentioned above.
Bootstrap has been used almost for everything: background, font-weight, colors, position, display, grid (rows and col's), margin, padding, height etc.. Except for specific html elements that required extra css styling like colors, font-family and also for hovering the elements.

The website replica contains at the top a navbar splitted in two parts, the header logo on top and  on bottom the nav elements along with the input filled (on the right).

The main header section is made out of three columns (feature stories, top stories and opinion), following on a long variety of different sections dedicated to main news themes like health, tech and science, latest news etc.
In the lat part of the website there is a dedicated section to media sharing and signing up, followed by the simplistic footer.
Thank you for reading.

### CODE REVIEW

**Comment by rammazzoti2000** [comment]
I know stickler is giving me this wired errors, but everything has been fixed, all the errors is showing I don't see them in my code, I don't know what's happening, if anyone has a suggestion why is this happening, is showing me code I don't have?!

----------------------------

Updated, saved by mistake the website newsweek as html format in one of my workflow folders, man  that was a big mistake, hopefully solved it now, sorry for that.

File: style/style.css [hunk 1]
```diff
@@ -0,0 +1,815 @@
...
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+/* Remove default padding */
+ul[class],
+ol[class] {
+  padding: 0;
```
Comment (mosesogwo) [inline style/style.css]: Replace with bootstrap class

File: style/style.css [hunk 2]
```diff
@@ -0,0 +1,815 @@
...
+  top: 0;
+  z-index: 999;
+  border-bottom: 2px solid rgba(0, 0, 0, 0.1);
+}
+
+.left-right-container {
+  padding: 0 10px;
+  background-color: #f72210;
+  color: white;
+  text-decoration: none;
```
Comment (mosesogwo) [inline style/style.css]: Replace all instances of text-decoration-none with bootstrap class .text-decoration-none

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,815 @@
...
+  color: red;
+  text-decoration: none;
+}
+
+h3.global-title a {
+  color: black;
+}
+
+.more-cards div a {
+  font-weight: bold;
```
Comment (mosesogwo) [inline style/style.css]: Use bootstrap class

File: style/style.css [hunk 4]
```diff
@@ -0,0 +1,815 @@
...
+  font-size: 1.7rem;
+  text-decoration: none;
+}
+
+.fa-bars {
+  display: none;
+}
+
+.header-bottom::before {
+  float: right;
```
Comment (mosesogwo) [inline style/style.css:169]: Use bootstrap class

File: style/style.css [hunk 5]
```diff
@@ -0,0 +1,815 @@
...
+  color: red;
+  text-decoration: none;
+}
+
+.news-navbar li a:hover {
+  color: red;
+}
+
+.social li {
+  display: inline-block;
```
Comment (mosesogwo) [inline style/style.css]: Use bootstrap class

File: style/style.css [hunk 6]
```diff
@@ -0,0 +1,815 @@
...
+
+main.container-fluid {
+  width: 96.9%;
+}
+
+.bottom-left {
+  position: absolute;
+  bottom: 0;
+  background: white;
+  text-align: center;
```
Comment (mosesogwo) [inline style/style.css]: Use bootstrap class

File: style/style.css [hunk 7]
```diff
@@ -0,0 +1,815 @@
...
+  font-size: 1.4rem;
+}
+
+div.thumbnail {
+  border: 0 none;
+  box-shadow: none;
+}
+
+h5.card-title {
+  margin-top: 0;
```
Comment (mosesogwo) [inline style/style.css]: Use bootstrap class

File: style/style.css [hunk 8]
```diff
@@ -0,0 +1,815 @@
...
+  max-width: 63px;
+}
+
+div.card-body {
+  padding: 0;
+}
+
+section.container-fluid {
+  padding: 0;
+}
```
Comment (mosesogwo) [inline style/style.css]: Use bootstrap classes

File: style/style.css [hunk 9]
```diff
@@ -0,0 +1,815 @@
...
+  left: 0;
+  z-index: 1;
+}
+
+.magazine-cover .img-wrapper .category {
+  position: absolute;
+  bottom: 0;
+  color: red;
+  background: rgba(255, 255, 255, 0.7);
+  font-weight: bold;
```
Comment (mosesogwo) [inline style/style.css]: Use bootstrap class

**Review by mosesogwo — CHANGES_REQUESTED** [review]
### Hi @rammazzoti2000, you have a great project :+1:. Just a few changes are required:

1. Set the background of the navbar so that it doesn't contrast with the background of the logo. You can use a color picker to identify the exact color on the logo background and use that as your nav's background.

2. The social media icons in the footer should be aligned to the left at the same breakpoint where the page switches to a single column. Yours stay on the right and moves to the left at another breakpoint.

3. Regular use of `!important` is bad css practice. Try to avoid them.

4. We can do more with bootstrap classes. The goal of this project is to use them. See comments on your stylesheet and replace all similar stylings with bootstrap classes. I know it may be tedious, but that is the essence of this particular project.




_**Kindly make these changes and submit another code review request.**_

**Cheers:smile:**

### Status: Changes requested.

File: style/style.css (hunk 4 above)
Comment (rammazzoti2000) [reply style/style.css:169]: isn't ::before selector only available in css? I am just asking since I wouldn't know how to replace that selector with html

**Review by nikasulo — APPROVED** [review]
# Congrats! @rammazzoti2000 
You have done a great job! :rocket: Please merge and keep moving forward

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**